quic/s2n-quic-core/src/recovery/hybrid_slow_start.rs

```diff
@@ -17,20 +17,47 @@ pub struct HybridSlowStart {
     pub(super) threshold: f32,
     max_datagram_size: u16,
     rtt_round_end_time: Option<Timestamp>,
+    use_hystart_plus_plus: bool,
+    pub(super) ss_growth_divisor: f32,
+    css_count: usize,
+    css_baseline_min_rtt: Duration,
+    css_threshold: f32,
 }
 
 /// Minimum slow start threshold in multiples of the max_datagram_size.
 /// Defined as "hystart_low_window" in tcp_cubic.c
 const LOW_SSTHRESH: u16 = 16;
+/// Factor for dividing the RTT to determine the threshold. Defined in tcp_cubic.c (not a constant)
+/// The same value is used in https://www.ietf.org/archive/id/draft-ietf-tcpm-hystartplusplus-04.html#name-algorithm-details
+const THRESHOLD_DIVIDEND: u32 = 8;
+
+//= https://tools.ietf.org/id/draft-ietf-tcpm-hystartplusplus-04.txt#section-4.3
```

Comment:
did you try running ./scripts/compliance to download the RFC text and add it to the PR?

Reply:
I think I did. please let me know if I miss something.

Reply:
there should be a file `s2n-quic/specs/tools.ietf.org/id/draft-ietf-tcpm-hystartplusplus-04.txt` created after you run the command that you can add to the PR

Reply:
Oh. sorry. I overlooked the file. pushed.